File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1149.rb
soma=0
ns = gets.chomp.split(' ')
a = ns[0].to_i
n = ns[1].to_i
i = 2
while(n<=0) do
    n = ns[i].to_i
    i+=1
end
for i in 0..n-1 do
    soma=soma+a+i
end
puts(soma)

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1149.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split(' ')

soma=0
a = parseInt(lines.shift())
n = parseInt(lines.shift())
while(n<=0){
    n = parseInt(lines.shift())
}
for(i=0;i<n;i++){
    soma+=a+i
}
console.log(soma)
